source/_components/sensor.pollen.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: "Pollen.com"
+description: "Instructions on how to use Pollen.com data within Home Assistant"
+date: 2018-01-10 19:20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: pollen.jpg
+ha_category: Health
+ha_release: 0.62
```

Comment:
Unfortunately we just passed the window for new features for 0.62. So this will make it into 0.63. Sorry about forgetting about the main PR.

Reply:
No worries! Since I thought this was done, I blew away the branch; will open a new PR. Thanks!